src/bin/club-story

```diff
@@ -178,13 +182,14 @@ const main = async () => {
             downloadFiles(story);
         }
         if (story && program.gitBranch) {
-            let slug = story.name.toLowerCase()
-                .replace(/\s/g, '-')
-                .replace(/[^a-z0-9-]/g, '')
-                .slice(0, 30);
-            let branch = `${story.story_type}-${story.id}-${slug}`;
-            debug('checking out git branch');
-            execSync('git checkout -b ' + branch);
+            createBranchFromStory(`${story.story_type}-${story.id}-`, story.name);
+        }
+        if (story && program.gitBranchStandard) {
+            if (!config.mentionName) {
+                console.error(
```

Comment:
Shouldn't we *not* continue to `createBranchFromStory` if this conditional is true?